avatars/processors/perturbation.py

```diff
@@ -8,6 +8,55 @@
 
 
 class PerturbationProcessor:
+    """Specifies the perturbation level of specified variables, 0 means no perturbation.
```

Comment:
The first line should summarize what the processor does. The current version isn't user friendly.